spotify/main.py

```diff
@@ -286,8 +312,8 @@ def main():
             },
         )
 
-        start = dt.datetime.now() - dt.timedelta(days=30)
-        end = dt.datetime.now()
+        start = start_datetime - dt.timedelta(days=30)
```

Comment:
do we need to hardcode the days here, isn't it just the defined start date?

Reply:
We have a longer range of 30 days (so the last month) in this case. Don't we want to keep that?

I'm thinking it could be

```python
end = end_datetime
start = end_datetime - dt.timedelta(months=1)
```

What do you think?

Reply:
performance doesn't consider dates at all, so we can ignore it here completely, it is always the same responce

Reply:
thx